### ISSUE TITLE
Accessibility Testing for Transition Experience, tbd, Transition Experience

### ISSUE THREAD

**Issue by cora-goldston-bah** [department-of-veterans-affairs/va.gov-team#97750]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@cora-goldston-bah

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

We did not find any color contrast issues. We used the VADS components for our product except for one element that we are using experimentally with Design System Council permission. We did not deviate from the standard VADS component colors, so we did not anticipate significant color contrast issues.

The experimental component we’re using is the “Time-sensitive benefit” banner that appears on some of our recommended benefit cards. For this experimental banner, we confirmed through WhoCanUse that the white text has a 13.6:1 contract ratio with the background, which uses the “vads-color-primary-darker” token from the VA Design System.

We used Colorblindly and WhoCanUse to confirm the color contrast ratios between foreground elements and background elements.

### Who completed the axe scans?

@cora-goldston-bah

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

We ran the axe DevTools extension for Google Chrome on each page of our product. The axe DevTools checks did not identify any automatic, guided, or manual-check issues for any of our pages.

Below are the axe DevTools results for each page of our product:
•	[Introduction page](https://staging.va.gov/benefit-eligibility-questionnaire/introduction): no issues detected
•	[Goals questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/goals): no issues detected
•	[Total length of service questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/service/time-served): no issues detected
•	[Currently serving questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/service/current): no issues detected
•	[Previous period of service questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/service/completed): no issues detected
•	[Time since separation questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/separation): no issues detected
•	[Character of discharge questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/discharge): no issues detected
•	[Disability rating questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/disability): no issues detected
•	[GI Bill questionnaire page](https://staging.va.gov/benefit-eligibility-questionnaire/gi-bill): no issues detected
•	[Review page](https://staging.va.gov/benefit-eligibility-questionnaire/review-and-submit): no issues detected
•	[Results page](https://staging.va.gov/benefit-eligibility-questionnaire/confirmation?benefits=GIB%2CECC%2CFHV%2CSVC%2CVRE%2CVSC%2CDHS%2CVAP%2CMHC%2CFMP%2CVAL%2CCOE%2CVAH%2CBUR%2CTGI): no issues detected

### How did axe testing go?

Our new “window shopping” page is still in development, so we were unable to test it with axe DevTools. However, we plan to add the window shopping page to the staging site before our code freeze.

### Who completed the content zoom and reflow test?

@cora-goldston-bah

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [ ] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

Everything was readable and usable at 200% zoom for all pages.

On the Results page, the filter and sort options disappeared completely at 300% zoom. There were no dropdown menus or links to allow the user to filter and sort. The filtering and sorting options reappeared at 400% zoom as a consolidated “filter and sort” link that works as expected. We will troubleshoot this issue.

At 400% zoom, we found some odd line breaks on the Introduction page. We will troubleshoot this issue.

### Who completed the keyboard navigation test?

@cora-goldston-bah

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

We tested each page of our product and we did not encounter any issues with keyboard navigation. All of the interactive components responded appropriately to the WebAIM expected keys. Every interactive element was able to receive keyboard focus and the order of [Tab] stops was logical.of this testing.

### Do you have any other results to share?

We do not have additional findings to share at this time. We will work with the Governance Team to complete advanced accessibility testing, as we do not have access to some assistive technology.

**Comment by cora-goldston-bah** [comment]
Hi @briandeconinck , we've completed the foundational accessibility testing on our staging site, in preparation for our December 6th Staging Review.

We only found two issues with zooming — one issue at 300% zoom and one at 400% zoom. Can you review our content zoom and reflow notes, and let us know if either of these issues will be launch-blocking at Staging Review?

Thank you!

cc: @nkontrabecki

**Comment by nkontrabecki** [comment]
@briandeconinck  Our team addressed the issue we were encountering for 300% zoom. Thank you for your prompt response. We look forward to your feedback at our Staging Review on 12/6/2024. Happy Thanksgiving!

cc @cora-goldston-bah
